CLUEstering/CLUEstering.py

```diff
@@ -13,10 +14,26 @@
 import pandas as pd
 from sklearn.datasets import make_blobs
 from sklearn.preprocessing import StandardScaler
-import CLUEsteringCPP as Algo
-
-
-def test_blobs(n_samples: int, n_dim: int, n_blobs: int = 4, mean: float = 0,
+from os.path import dirname, join
+path = dirname(__file__)
+import sys
+sys.path.insert(1, join(path, 'lib'))
+import CLUE_Convolutional_Kernels as clue_kernels
+import CLUE_CPU_Serial as cpu_serial
+tbb_found = False
+if len(glob(join(path, 'lib/CLUE_CPU_TBB*.so'))) != 0:
```

Comment:
Maybe:
```suggestion
tbb_found = os.path.exists(join(lib_path, 'CLUE_CPU_TBB*.so'))
if(tbb_found):
```

And same below?

Reply:
Maybe we should add the `else` statement, telling the user that the backend has not been loaded?

Reply:
Ok perfect :)

Reply:
> Maybe:
> 
> And same below?

Yes we can change it, but we still need to glob, otherwise it doesn't find the path

Reply:
```
tbb_found = exists(*glob(join(path, 'lib/CLUE_CPU_TBB*.so')))
```